### ISSUE TITLE
daily builds are missing pandoc

### ISSUE THREAD

**Issue by kevinushey** [rstudio/rstudio#6808]
### System details

    RStudio Edition : Desktop
    RStudio Version : v1.4.299
    OS Version      : Windows 10 x64
    R Version       : 4.0.0

### Steps to reproduce the problem

Install RStudio, and run:

```
pandoc <- Sys.getenv("RSTUDIO_PANDOC")
pandoc
file.exists(pandoc)
```

### Describe the problem in detail

```
> pandoc <- Sys.getenv("RSTUDIO_PANDOC")
> pandoc
[1] "C:/Program Files/RStudio/bin/pandoc"
> file.exists(pandoc)
[1] FALSE
```

### Describe the behavior you expected

Pandoc should be bundled with RStudio.

- [x] I have read the guide to submitting good bug reports at https://github.com/rstudio/rstudio/wiki/Writing-Good-Bug-Reports . 
- [x] I have installed the latest version of RStudio and confirmed that the issue still persists.
- [x] If I am reporting a RStudio crash, I have included a diagnostics report. https://support.rstudio.com/hc/en-us/articles/200321257-Running-a-Diagnostics-Report
- [x] I have done my best to include a minimal, self-contained set of instructions for consistently reproducing the issue.

**Comment by jmcphers** [comment]
From the logs:

```
2020-05-04 13:41:25  2020-05-04 20:41:25 (17.9 MB/s) - 'pandoc-2.9.2.1-windows-x86_64.zip' saved [36379904/36379904]
2020-05-04 13:41:25  
2020-05-04 13:41:25  Unzipping pandoc-2.9.2.1-windows-x86_64.zip
2020-05-04 13:41:26  The system cannot find the path specified.
2020-05-04 13:41:26  The system cannot find the file specified.
```

This is probably the part of the install process that's failing:

https://github.com/rstudio/rstudio/blob/f20a883035522081bdd5a7e5afbffdecdc89c1a2/dependencies/windows/install-dependencies.cmd#L134-L142

Very likely Pandoc has decided to change their folder structure again.

**Comment by R-Hannibal** [comment]
Verified Fixed on: RStudio Version - 1.4.753, R Version - 4.0.2, Windows 10 x64 - Desktop 

Pandoc is present after installing the latest Daily build..
